package.json

```diff
@@ -36,6 +36,7 @@
     ]
   },
   "dependencies": {
+    "@types/analytics-node": "^3.1.9",
```

Comment:
Should it be actually in the frontend/package.json?

Reply:
yep

Reply:
Types probably should go into `devDependecies` section.